cluster-autoscaler/processors/podinjection/pod_injection_processor_test.go

```diff
@@ -419,3 +447,9 @@ func withNodeName(nodeName string) podOption {
 		pod.Spec.NodeName = nodeName
 	}
 }
+
+func withCreationTimestamp(timestamp time.Time) podOption {
```

Comment:
Can you add this function to https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/utils/test/test_utils.go instead? Existing pod options from here can also be migrated, `withNodeName` is actually already there.